config/config.exs

```diff
@@ -28,3 +28,5 @@ use Mix.Config
 # here (which is why it is important to import them last).
 
 import_config "#{Mix.env()}.exs"
+
+config :tesla, adapter: Tesla.Adapter.Hackney
```

Comment:
Maybe this will change when resolving #14

Reply:
As @britto said I think we can have both _options_, application environments and function injected configurations 👀

Reply:
Yes, I'm thinking in resolve all these config related stuff in the #14 😄